### PR TITLE
Update button styling for new experiment and new run

### PR DESCRIPTION
**Pull request by rileyjbauer**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/264)

### CODE REVIEW

File: frontend/src/components/Toolbar.tsx [hunk 1]
```diff
@@ -155,7 +156,8 @@ class Toolbar extends React.Component<ToolbarProps> {
               <div>{/* Extra level needed by tooltip when child is disabled */}
                 <BusyButton id={b.id} color='secondary' onClick={b.action} disabled={b.disabled}
                   title={b.title} icon={b.icon} busy={b.busy || false}
-                  outlined={b.outlined || false} />
+                  outlined={b.outlined && !b.primary || false}
```
Comment (yebrahim) [inline frontend/src/components/Toolbar.tsx]: Can we add parentheses here? Off the top of my head, I don't know what this evaluates to. :)

Comment (rileyjbauer) [reply frontend/src/components/Toolbar.tsx]: Good call

File: frontend/src/components/Toolbar.tsx [hunk 2]
```diff
@@ -155,7 +156,8 @@ class Toolbar extends React.Component<ToolbarProps> {
               <div>{/* Extra level needed by tooltip when child is disabled */}
                 <BusyButton id={b.id} color='secondary' onClick={b.action} disabled={b.disabled}
                   title={b.title} icon={b.icon} busy={b.busy || false}
-                  outlined={b.outlined || false} />
+                  outlined={(b.outlined && !b.primary) || false}
```
Comment (yebrahim) [inline frontend/src/components/Toolbar.tsx]: Why is the `|| false` needed here?

**Pull request merged**